src/containers/App/App.js

```diff
@@ -20,7 +20,10 @@ class App extends Component {
     Sentry.init({
       dsn: "https://a5fc08e12a1744ddacc396ce79e034f2@sentry.io/1462422",
     });
-    ReactGA.initialize('UA-140402020-1'); // Google Analytics
+    // Google Analytics
+    ReactGA.initialize('UA-140402020-1', {
+      testMode: process.env.NODE_ENV === 'test',
```

Comment:
@boralben this comes from here: https://github.com/react-ga/react-ga/issues/257

Webpack is supposed to set NODE_ENV correctly and inject it as `production` (or prod or whatever) in the final build, but I'm not sure how to test that this actually works as intended and disables test mode when not testing. Let me know what you think